front/src/applications/stdcm/types.ts

```diff
@@ -125,3 +145,15 @@ export enum StdcmStopTypes {
   DRIVER_SWITCH = 'driverSwitch',
   SERVICE_STOP = 'serviceStop',
 }
+
+export type StdcmLinkedPathStep = {
+  date: string;
+  time: string;
+  isoArrivalTime: string;
+} & SearchResultItemOperationalPoint;
```

Comment:
Give only the fields you will need
```suggestion
  trigram: string;
};
```